src/ros_license_toolkit/checks.py

```diff
@@ -332,6 +330,21 @@ def _evaluate_result(self, package: Package) -> None:
                 self.files_not_matched_by_any_license_tag,
                 package,
             )
+        elif self.files_with_inofficial_tag:
+            info_str = ''
+            info_str += 'For the following files, please change the ' +\
+                'License Tag in the package file to SPDX format:\n' +\
+                '\n'.join(
+                    [f"  '{x[0]}' is of {x[1][0]} but its Tag is {x[1][1]}."
+                     for x in self.files_with_inofficial_tag.items()])
+            self._warning(info_str)
+        elif self.files_with_no_licenses:
+            info_str = ''
+            info_str += 'For the following files, please add ' +\
+                'a License Header in SPDX format:\n' +\
+                '\n'.join(
+                    [f"  {x}" for x in self.files_with_no_licenses])
+            self._warning(info_str)
```

Comment:
@ct2034 Do you think a code file without license text should trigger a warning or even an Error?

Reply:
Do you mean any file in the package where there was no license information found inside?
I think this can be very common, for example in configuration files. I think this could be just ignored. Unless we can be sure it is a source file. Then it should maybe be an error.